src/components/Transaction.js

```diff
@@ -1,14 +1,12 @@
 import React from 'react'
 
-const Transaction = () => {
+export default function Transaction(transaction) {
   return (
```

Comment:
if you like, you can avoid repeated the word `props` by pulling the transaction off of the component. `const {transaction} = props`